example/traffic_mgmt/Makefile.am

```diff
@@ -1,9 +1,5 @@
 include $(top_srcdir)/example/Makefile.inc
 
-bin_PROGRAMS = odp_traffic_mgmt$(EXEEXT)
-odp_traffic_mgmt_LDFLAGS = $(AM_LDFLAGS) -static
-odp_traffic_mgmt_CFLAGS = $(AM_CFLAGS) -I${top_srcdir}/example
+bin_PROGRAMS = odp_traffic_mgmt
```

Comment:
@lumag  I still do not understand do you remove extensions? That link http://www.gnu.org/software/automake/manual/html_node/EXEEXT.html says that there is no way to do it anyhow different. I think removing will break existence functionality.

Reply:
What for? `_PROGRAMS` will automatically receive `$(EXEEXT)` if necessary, so do `$(TESTS)`. What purpose do these `$(EXEEXT)` surve at this point?

Reply:
point is to name programs with extension like test.exe and run this under make check. Commit 0274087a added that.

Reply:
@muvarov The mentioned commit is plain wrong, starting from the `AC_SUBST([EXEEXT])`, which is not necessary. All `_PROGRAMS` variables will be automatically extended with `$(EXEEXT)` during `Makefile.am` -> `Makefile.in` conversion by Automake, it is stated in the documentation. In our case this suffix is only necessary if `dirA` uses (in `TESTS` variable) program built in `dirB`. For example, like `test/linux-generic/Makefile.am` does for `validation/api/shmem/shmem_linux` or `ring/ring_main`. Oh, wait. Where is the `$(EXEEXT)` suffix in the former case?

Reply:
@lumag now I a little bit lost. Lets sync up on monday about that.